### PR TITLE
Update faqs.mdx

### LINKED ISSUE #3111: [DOCS] Wrong links in the FAQ
### Description

There are various broken links on the FAQ page (such as the single-user mode or Hacktoberfest pages) which should be fixed immediately.

_Steps to contribute_

1. Fork the repository [here](https://github.com/EddieHubCommunity/LinkFree/fork).
2. Create a new branch named `fix_faq_links`.
3. Open the `faqs.mdx` file, which is located at `pages/docs/`.
4. In line 9, change the single-user mode link to `advanced/single-user-mode`. We do not want to include `docs` as otherwise, it would lead to `docs/docs/...` which is invalid.
5. In line 15, change the Hacktoberfest link to `contributing/hacktoberfest`.
6. The Discord invitation link seems to be broken as well. Change it to `http://discord.eddiehub.org` (located in line 21).
7. Commit and push your changes.
8. Create a [pull request](https://github.com/EddieHubCommunity/LinkFree/compare).
9. You're done now! 🎉 You just have to wait until the maintainers check your PR.

If you need any help, let us know about it and we'll gladly help you. 🙂

### Screenshots

_No response_

### Additional information

This is a good first issue. Please leave it open to first-time contributors and beginners.

### PR DESCRIPTION
**Pull request by Rishabh-Singh-Codes**
## Fixes Issue
Closes #3111

## Changes proposed
Updated the links as described.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3116"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Awesome. Thanks! 🚀

**Comment by HarshDeep61034** [comment]
Looks Good, great @Rishabh-Singh-Codes

File: pages/docs/faqs.mdx [hunk 1]
```diff
@@ -6,19 +6,19 @@ It is completely free to use LinkFree.
 
 ## Is it all Open Source?
 
-Yes, LinkFree is 100% Open Source. You can use our hosted version, or you can host it yourself in [Single User Mode](docs/single-user-mode).
+Yes, LinkFree is 100% Open Source. You can use our hosted version, or you can host it yourself in [Single User Mode](advanced/single-user-mode).
```
Comment (eddiejaoude) [inline pages/docs/faqs.mdx:9]:
```suggestion
Yes, LinkFree is 100% Open Source. You can use our hosted version, or you can host it yourself in [Single User Mode](/advanced/single-user-mode).
```

File: pages/docs/faqs.mdx [hunk 2]
```diff
@@ -6,19 +6,19 @@ It is completely free to use LinkFree.
...
 
 We recommend using the hosted version here, so that your profile is more discoverable and you don't have to do the upgrades and updates yourself.
 
 ## Does this count towards Hacktoberfest?
 
-See [Hacktoberfest](docs/hacktoberfest) page.
+See [Hacktoberfest](contributing/hacktoberfest) page.
```
Comment (eddiejaoude) [inline pages/docs/faqs.mdx:15]:
```suggestion
See [Hacktoberfest](/docs/contributing/hacktoberfest) page.
```

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you 👍 I left some inline comments

File: pages/docs/faqs.mdx (hunk 1 above)
Comment (Panquesito7) [reply pages/docs/faqs.mdx:9]: Not sure if it should have `docs` on it? I tried that on Gitpod and it lead to `docs/docs`.
As you're already on the docs page, it'll redirect you to `advanced/single-user-mode`.

I also saw other docs pages do this (without the `docs`) and work.

**Comment by kumarsonsoff3** [comment]
Seems your branch is out of date leading to the merge conflicts, try updating your branch and then make the requested changes. Let us know if you need any help with this!

File: pages/docs/faqs.mdx (hunk 1 above)
Comment (eddiejaoude) [reply pages/docs/faqs.mdx:9]: It shouldn't matter where someone is, if all links start with `/`

**Comment by eddiejaoude** [comment]
I will merge to a temporary branch and fix, as this is getting out of date and causing conflicts

**Pull request merged**